pkg/mountmanager/fake-safe-mounter.go

```diff
@@ -103,6 +103,14 @@ func (f *FakeNodeMounterWithCustomActions) MakeFile(pathname string) error {
 	return nil
 }
 
+// Resize returns boolean and error if any
+func (f *FakeNodeMounterWithCustomActions) Resize(devicePath string, deviceMountPath string) (bool, error) {
```

Comment:
This code would work for `FakeNodeMounter` which does not use fake actions, but doing this would result in avoiding the fake actions completely. I think this code should mimic the real code so it actually uses the fakes.

Maybe I'm missing something, do we now intend to get rid of the fakes and override `Resize()` like this?

Reply:
@RomanBednar  Could you verify the changes now

Reply:
@ambiknai lgtm